src/main/scala/scalaz/nio/channels/AsynchronousChannel.scala

```diff
@@ -211,21 +211,21 @@ object AsynchronousChannel {
     IO.async0[Exception, T] { k =>
```

Comment:
I think this could be done using `IO.asyncInterrupt` but I couldn't make it work.

Reply:
Is `IO.asyncInterrupt` new in `0.5.3`? I mean is it necessary for the library upgrade PR?

Reply:
I don't think it's new in `0.5.3` and no it's not needed for the upgrade.